docs/docs/themes.md

```diff
@@ -16,6 +16,5 @@ This means that the configuration and functionality isn’t directly written int
 
 ## Other resources
 
+- [Gatsby theme tutorials](/tutorial/theme-tutorials/)
```

Comment:
It looks like there are some unrelated docs changes here. Ok to remove them?

Reply:
Oops! I think this is because I made a new branch from master, but this PR is against the `feat/gatsby-source-v4` branch, not against master, and that branch was made about a month ago!

Reply:
Hmm, I'm not sure how to best remove them. I updated the `feat/gatsby-source-wordpress-v4` branch from master so those commits should exist on that branch, but they still show up in this PR for some reason. I'm worried if I remove them, it will eventually remove them from master when `feat/gatsby-source-wordpress-v4` makes it to master.